### ISSUE TITLE
DeepSort algorithm的链接找不到了

### ISSUE THREAD

**Issue by sunfeng90** [LZQthePlane/Online-Realtime-Action-Recognition-based-on-OpenPose#61]
你好，你做的很棒。想问一下，DeepSort algorithm是哪个算法？对应的链接找不到内容了。是TianzongSong的Real-Time-Action-Recognition项目中用到的sort算法吗?

**Comment by ajithvcoder** [comment]
@sunfeng90 Deep sort is a multi object multi instance tracking algorithm .  https://github.com/nwojke/deep_sort    https://arxiv.org/abs/1703.07402

**Comment by sunfeng90** [comment]
@ajithvallabai Thanks.Do you know how to see the details of   the  VGG_origin model and mobilenet_thin?

**Comment by ajithvcoder** [comment]
@sunfeng90 What details ? do u want to set the details  ?

**Comment by sunfeng90** [comment]
> @sunfeng90 What details ? do u want to set the details ?

@ajithvallabai the code of   models. Do  you  know it?

**Comment by ajithvcoder** [comment]
@sunfeng90  yes models are present here Online-Realtime-Action-Recognition-based-on-OpenPose\Pose\graph_models 
and its loaded in main.py

**Comment by pu8019** [comment]
您好，我想知道还有下载 ./download.sh的路径吗，有的话能给我发一下吗
